alot/ui.py

```diff
@@ -1,9 +1,10 @@
 # Copyright (C) 2011-2012  Patrick Totzke <patricktotzke@gmail.com>
 # This file is released under the GNU GPL, version 3 or a later revision.
 # For further details see the COPYING file
-import urwid
 import logging
+import os
```

Comment:
I'd much rather have this os and XDG related stuff in `alot.init` than here. The UI class is too big already anyway..